CHANGELOG.md

```diff
@@ -9,7 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [16.6.9] - 2023-12-22
 
--   Add session util functions `withSession` and `getSSRSession` for Next.js App directory.
+-   Add session util functions `withSession`, `getSSRSession` and `withPreParsedRequestResponse` for Next.js App directory.
```

Comment:
this should probably be a minor version update cause these are new features / functions being added